.github/workflows/recipe-catalog-change.yaml

```diff
@@ -0,0 +1,222 @@
+name: Run recipe tests on catalog change
+
+on:
+  push:
+    paths:
+      - 'packages/backend/src/assets/ai.json'
+  pull_request:
+    paths:
+      - 'packages/backend/src/assets/ai.json'
+
+jobs:
+  windows:
+    name: windows-${{ matrix.windows-version }}-${{ matrix.windows-featurepack }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      checks: write
+    env:
+      MAPT_VERSION: v0.6.8
+      MAPT_IMAGE: quay.io/redhat-developer/mapt
+    strategy:
+      fail-fast: false
+      matrix:
+        windows-version: ['10','11']
+        windows-featurepack: ['22h2-ent', '23h2-ent']
+        exclude:
+        - windows-version: '10'
+          windows-featurepack: '23h2-ent'
+        - windows-version: '11'
+          windows-featurepack: '22h2-ent'
+
+
+    steps:
+    - name: Get Podman version used by Desktop
+      run: |
+        version=$(curl https://raw.githubusercontent.com/containers/podman-desktop/main/extensions/podman/packages/extension/src/podman5.json | jq -r '.version')
+        echo "Default Podman Version from Podman Desktop: ${version}"
+        echo "PD_PODMAN_VERSION=${version}" >> $GITHUB_ENV
+    - name: Set the default env. variables
+      env:
+        DEFAULT_FORK: 'containers'
+        DEFAULT_BRANCH: 'main'
+        DEFAULT_EXT_TESTS: '1'
+        DEFAULT_NPM_TARGET: 'test:e2e'
+        DEFAULT_ENV_VARS: 'TEST_PODMAN_MACHINE=true,ELECTRON_ENABLE_INSPECT=true'
+        DEFAULT_PODMAN_OPTIONS: 'INIT=1,START=1,ROOTFUL=1,NETWORKING=0'
+        DEFAULT_EXT_REPO_OPTIONS: 'REPO=podman-desktop-extension-ai-lab,FORK=containers,BRANCH=main'
```

Comment:
You still wont be able to test the change on PR with this setup.

Reply:
I believe I've fixed it